ui/admin/tests/acceptance/host-catalogs/host-sets/selectors.js

```diff
@@ -3,13 +3,16 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-export const FIELD_PREFERRED_ENDPOINT = '[name="preferred_endpoints"] input';
+export const FIELD_PREFERRED_ENDPOINT = '[name=preferred_endpoints] input';
 export const FIELD_PREFERRED_ENDPOINT_ADD_BTN =
-  '[name="preferred_endpoints"] button';
-export const FIELD_FILTERS = '[name="filters"] input';
-export const FIELD_FILTERS_ADD_BTN = '[name="filters"] button';
-export const FIELD_SYNC_INTERVAL = '[name="sync_interval_seconds"]';
-export const FIELD_AZURE_FILTER = '[name="filter"]';
+  '[name=preferred_endpoints] button';
+export const FIELD_PREFERRED_ENDPOINT_DELETE_BTN =
+  '[name=preferred_endpoints] [data-test-remove-button]';
+export const FIELD_FILTERS = '[name=filters] input';
+export const FIELD_FILTERS_ADD_BTN = '[name=filters] button';
+export const FIELD_SYNC_INTERVAL = '[name=sync_interval_seconds]';
+export const FIELD_AZURE_FILTER = '[name=filter]';
+export const FIELD_FILTERS_REMOVE = '[name=filters] [data-test-remove-button]';
```

Comment:
small nitpick: should this be renamed to `FIELD_FILTERS_REMOVE_BTN`

Reply:
💯 Nice catch!!! 😉